_episodes/22.md

```diff
@@ -0,0 +1,166 @@
+is now open! 
+
+### LinkedIn is hiring!
+
+* [Software Engineer - Big Data Platform](https://www.linkedin.com/jobs/view/2402727250/?alternateChannel=search&refId=VRDXEQNgS2gxtpsJaHPXjQ%3D%3D&trackingId=0GzsJkrXWYt6qHWSUHTvCg%3D%3D&trk=d_flagship3_search_srp_jobs)
+* [Senior Software Engineer - Big Data Platform](https://www.linkedin.com/jobs/view/2291645936/?eBP=CwEAAAF6y0tYtsROpAG7XxMEhLVgpq2rSMwpNv28Q_j06PdFsD_s11eFyh-sIv2rxm_Y8zN-p755Gts-ElMlR6XvK2hOMp3JMnxFPzOnZvvZnv_-oHaBslitgtWzsmJy7_f7BKljmgAUtfinG9WCp1Bpi574HZEBJwAsjzKx-89NUdnIBj_SBIPHES_G2RNqoKp5eZ4c0k7YaVJSuZJTyi2K6KoKJ7njT65FEOWvmS9S80ysbINbXjX_WSz71RNAugEpqIgE9-gB1MhW8tQ9z72jQhbjXMqSuUaYS43zFaP8ImXhjTrhbopTxyxTIN9yst6tvlcPo_T5RNAaf_0e8x_km2SGdw&recommendedFlavor=IN_NETWORK&refId=VRDXEQNgS2gxtpsJaHPXjQ%3D%3D&trackingId=5Qo2D07i3Wl%2FVhGeAvLtew%3D%3D&trk=flagship3_search_srp_jobs)
+
+
+## Concept of the week: Trino at LinkedIn
+
+The LinkedIn team covers the concept of the week in [this section]().
+
+## PR of the week: Digging into join queries
+
+This weeks PR of the week is a PR of the future! 🔮 We call this a future PR 
+because [LinkedIn is currently investigating the issue](https://github.com/jitheshtr/trino/issues/1).
+This gives us a chance to talk about the research aspects that go into a PR.
+
+With a view `V` that performs a `UNION ALL` from an old table `O` and a new 
+migrated table `N`. For `datepartition` values older than `D` (say 2021-06-05), 
+table `O` will be referred for data, while for date equal to or greater than `D`,
+data from `N` will be used.
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/22/view-old-new-tables.png"/><br/>
+Commander Bun Bun, landing the job!
+</p>
+
+The query in question is:
+
+```
+SELECT * FROM V
+WHERE x IN (SELECT x2 FROM Z)
+AND cast(substring(datepartition,1,10) as date) >= date('2021-06-08')
+```
+
+Here, table `Z` has stats available and only have 17 rows in them. While the 
+data from view `V` (which is entirely from underlying table `N` for this query) 
+has say billions of rows.
+
+This query used to take about 39 seconds to run before our upgrade 
+(PrestoSQL-333). After the upgrade (Trino-352) it increased to approximately 
+thirty-five minutes.
+
+## Demo: How to research the performance of a join
+
+To research these issues, see the [steps taken to investigte the issue](https://github.com/jitheshtr/trino/issues/1).
+
+## Question of the week: How can I query the Hive views from Trino?
+
+We actually covered the answer in [episode 18](https://trino.io/episodes/18.html).
+
+You can use the [Coral](https://engineering.linkedin.com/blog/2020/coral) 
+project that allows for translation between different SQL syntax. For example, 
+it processes Hive QL statements and convert them to an internal representation using
+[Apache Calcite](https://calcite.apache.org/). It then converts the internal
+representation to Trino SQL.
+
```

Comment:
Somewhere in here link to the actual docs https://trino.io/docs/current/connector/hive.html#hive-views